app/admin/golem/components/CategoryBadge.tsx

```
@@ -0,0 +1,58 @@
+    await onSave(nextValue);
+    setSaving(false);
+    setEditing(false);
+  };
+
+  if (editing) {
+    return (
+      <div className="flex items-center gap-1">
+        <select
+          value={displayValue}
+          onChange={(event) => handleSave(event.target.value)}
+          autoFocus
```

Biome (2.3.13)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)